javarosa/util/schema-gen/src/org/javarosa/engine/FunctionExtensions.java

```diff
@@ -0,0 +1,54 @@
+package org.javarosa.engine;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.condition.IFunctionHandler;
+
+import java.util.Date;
+import java.util.Vector;
+
+/**
+ * Custom functions to be used in form player debug tools.
+ *
+ * Allows users to:
+ *  - override 'today()' and 'now()' with custom dates.
```

Comment:
Out of scope, but it'd be nice to be able to optionally pass a full DateTime into this function since that's what now() should be returning